mesonbuild/ast/printer.py

```diff
@@ -133,7 +133,7 @@ def visit_AndNode(self, node: mparser.AndNode) -> None:
 
     def visit_ComparisonNode(self, node: mparser.ComparisonNode) -> None:
         node.left.accept(self)
-        self.append_padded(node.ctype, node)
+        self.append_padded(node.ctype if node.ctype != 'notin' else 'not in', node)
```

Comment:
I think this is papering over the actual issue, that mparser seems to have the `'notin'` and `'not in'` switched in the comparison map?

Reply:
`notin` is the node type when parser found `not` token followed by `in` token.